cmake/OpenCVFindLibsGUI.cmake

```diff
@@ -60,6 +60,7 @@ if(WITH_QT)
 endif()
 
 # --- OpenGl ---
+set(OpenGL_GL_PREFERENCE LEGACY)
```

Comment:
`ocv_update` is preferable to allow external control.

Reply:
fixed.